core/src/client/lwm2m_static.c

```diff
@@ -599,4 +599,53 @@ AwaError AwaStaticClient_RegisterResourceWithHandler(AwaStaticClient * client, c
                                             NULL, 0 , 0);
 }
 
+AwaError AwaStaticClient_ResourceChanged(AwaStaticClient * client, AwaObjectID objectID, AwaObjectInstanceID objectInstanceID, AwaResourceID resourceID)
+{
+    AwaError result = AwaError_Unspecified;
+
+    if (client != NULL)
+    {
+        ResourceDefinition * definition = Definition_LookupResourceDefinition(Lwm2mCore_GetDefinitions(client->Context), objectID, resourceID);
+        if (definition != NULL)
+        {
+            if (definition->Handlers.Read == NULL)
+            {
+                if (definition->Handler != NULL)
+                {
+                    void * valueBuffer = NULL;
+                    int valueBufferSize = 0;
+                    int resourceInstanceID = 0;  // FIXME
```

Comment:
We currently do not support write attributes on multiple instance resources either, but this should work for all single instance resources